server/src/main/java/io/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -270,4 +264,19 @@ public void cleanupCacheFiles(File baseFile, File cacheFile) throws IOException
       }
     }
   }
+
+  public Set<StorageLocation> getSortedSet(List<StorageLocation> locs)
```

Comment:
Is StorageLocation in `locs` something that is intended to happen, or would that scenario be a bug?